From 6968e0787e2269ce45d46e5a55f09803fc25ba14 Mon Sep 17 00:00:00 2001
From: luca020400 <luca.stefani.ge1@gmail.com>
Date: Wed, 15 Oct 2014 18:59:23 +0200
Subject: [PATCH 1/1] reduce scan timeout to workaround ar6000

---
 src/drivers/driver_wext.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/drivers/driver_wext.c b/src/drivers/driver_wext.c
index 1401050..93f7254 100644
--- a/src/drivers/driver_wext.c
+++ b/src/drivers/driver_wext.c
@@ -1058,7 +1058,7 @@ int wpa_driver_wext_scan(void *priv, struct wpa_driver_scan_params *params)
 		 * when scan is complete, so use longer timeout to avoid race
 		 * conditions with scanning and following association request.
 		 */
-		timeout = 30;
+		timeout = 5;
 	}
 	wpa_printf(MSG_DEBUG, "Scan requested (ret=%d) - scan timeout %d "
 		   "seconds", ret, timeout);
-- 
1.9.1
